sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 - Removed type `ExtractKeyPhrasesOptions`, `RecognizeEntitiesOptions`, `RecognizeLinkedEntitiesOptions` and respective exposures.
 - Removed the property `statistics` from `AnalyzeActionsResult` as it is not currently returned by the service even if 
   the user passes `includeStatistics` = `true`.
+- Removed constructors, but to use the private setter accessor to assign the additional properties:
```

Comment:
Were these exposed in the earlier betas?

Reply:
Yes. Only in betas.